tests/integration/block_sdk_suite.go

```diff
@@ -109,7 +109,8 @@ func (s *IntegrationTestSuite) SetupSubTest() {
 	// query height
 	height, err := s.chain.(*cosmos.CosmosChain).Height(context.Background())
 	require.NoError(s.T(), err)
-	WaitForHeight(s.T(), s.chain.(*cosmos.CosmosChain), height+1)
+	WaitForHeight(s.T(), s.chain.(*cosmos.CosmosChain), height+3)
+	s.T().Logf("reached height %d", height+2)
```

Comment:
difference between the heights here.

want to just add the logging to `WaitForHeight()`?